src/helpers.sh

```diff
@@ -100,3 +100,13 @@ function helper::unsetIfExists() {
     unset "$function_name"
   fi
 }
+
+helpers::find_files_recursive() {
+    local path="$1"
+
+    if [[ -d "$path" ]]; then
+        find "$path" -type f -name '*test.sh' | sort | uniq
```

Comment:
```suggestion
        find "$path" -type f -name '**/*test.sh' | sort | uniq
```

Reply:
Not needed. The `find ... -name '*test.sh' ` is already looking at all files that has a name ending with `test.sh`, so we don't need to use the `**/*test.sh`. Or did you experience any issue testing it on Windows/Linux, @Tito-Kati ?

Reply:
Only the ones directly under that directory, do we want recursive search?

Reply:
Isn’t `find` already searching recursively?
<img width="754" alt="Screenshot 2023-10-03 at 15 18 27" src="https://github.com/TypedDevs/bashunit/assets/5256287/28f96643-22aa-46d1-a168-367495a5c631">

Reply:
You are right xD